Makefile

```
@@ -710,6 +714,56 @@ install-node: $(HOME)/opt ## Install the Node.js runtime.
 		tar xf "$${tempfile}"; \
 		ln -sf node-v$(NODE_VERSION)-linux-x64 node
 
+	@python -m venv .venv
+
+.PHONY: install-python
+install-python: $(HOME)/.pyenv/versions/ian/.installed ## Install the Python environment.
```

Create a config variable for PYENV_ROOT
```suggestion
install-python: $(PYENV_ROOT)/versions/ian/.installed ## Install the Python environment.
```